bootloader/lkenv/lkenv_test.go

```diff
@@ -461,6 +462,17 @@ func (l *lkenvTestSuite) TestLoadValidatesVersionSignatureConsistency(c *C) {
 	}
 }
 
+func (l *lkenvTestSuite) TestLoadPropagatesErrNotExist(c *C) {
+	// make sure that if the env file doesn't exist, the error returned from
+	// Load() is os.ErrNotExist, even if it isn't exactly that
+	env := lkenv.NewEnv("some-nonsense-file-this-doesnt-exist", lkenv.V1)
+	c.Check(env, NotNil)
+
+	err := env.Load()
+	c.Assert(xerrors.Is(err, os.ErrNotExist), Equals, true, Commentf("err is %+v", err))
```

Comment:
Failing with go 1.9:
```

----------------------------------------------------------------------
FAIL: lkenv_test.go:465: lkenvTestSuite.TestLoadPropagatesErrNotExist

lkenv_test.go:472:
    c.Assert(xerrors.Is(err, os.ErrNotExist), Equals, true, Commentf("err is %+v", err))
... obtained bool = false
... expected bool = true
... err is cannot open LK env file:
    github.com/snapcore/snapd/bootloader/lkenv.(*Env).LoadEnv
        github.com/snapcore/snapd/bootloader/lkenv/_test/_obj_test/lkenv.go:184
  - open some-nonsense-file-this-doesnt-existbak: no such file or directory
```

Reply:
I added a workaround for go 1.9 to this by manually wrapping the error with a custom type that implements Unwrap() and returning os.ErrNotExist on Go 1.9 when we know the error for sure is os.ErrNotExist